docs/user_guide.md

```diff
@@ -595,6 +596,16 @@ optional arguments:
                         reported defect.
   --verbose {info,debug,debug_analyzer}
                         Set verbosity level. (default: info)
+
+export arguments:
+  -e {html}, --export {html}
```

Comment:
What do you think about the `--format` argument, instead of the `--export`?

Reply:
`--format` is only logical if `--format html` makes `parse` output **only** HTML, and no `stdout` text. Similarly, `--format plaintext` (the default value) should result in a console output.

But then, the format specifier optstring should be named `-o/--output`, as in `cmd` and `analyze`.